README.md

```diff
@@ -102,7 +94,7 @@ Origins = https://foreman.demo.lan
 Action = remote-login-ssh
 
 [SSH-Login]
-command = /opt/theforeman/tfm/root/usr/share/gems/gems/foreman_remote_execution-1.8.0/extra/cockpit/foreman-cockpit-session
+command = /usr/sbin/foreman-cockpit-session
```

Comment:
This also requires Foreman 1.24, right?

Reply:
Yes, also it does not work yet :-) https://bugzilla.redhat.com/show_bug.cgi?id=1770999 - I need to fix this. Domain cockpit_ws_t is not allowed to execute usr_t. I think the cleanest approach is to label this file as cockpit_session_exec_t, can you confirm please:

https://github.com/fedora-selinux/selinux-policy-contrib/blob/rawhide/cockpit.te#L49

Thanks.

Reply:
Or I wonder if I should label the file as cockpit_session_exec_t so the process is transitioned into cockpit_session_t which has slightly different rules:

https://github.com/fedora-selinux/selinux-policy-contrib/blob/rawhide/cockpit.te#L32

Then I will probably need to add couple of new allow rules in our policy for this domain as the script (a) reads its configuration from a YAML file and (b) reaches out to Foreman using HTTPS. These are the major things I see in the source code (need to test for more).

Just please confirm is this is the SELinux domain that is supposed to be used. Thanks.